src/main/java/org/opentripplanner/api/resource/BenchmarkDominanceResource.java

```diff
@@ -0,0 +1,85 @@
+package org.opentripplanner.api.resource;
+
+import org.glassfish.grizzly.http.server.Request;
+import org.opentripplanner.api.common.RoutingResource;
+import org.opentripplanner.routing.core.RoutingRequest;
+import org.opentripplanner.routing.impl.GraphPathFinder;
+import org.opentripplanner.routing.spt.GraphPath;
+import org.opentripplanner.scripting.api.BenchmarkResult;
+import org.opentripplanner.standalone.Router;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.UriInfo;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+import static org.opentripplanner.api.resource.ServerInfo.Q;
+
+//Used for benchmarking router.
+@Path("routers/{routerId}/benchmark")
+public class BenchmarkDominanceResource extends RoutingResource {
+    private static final Logger LOG = LoggerFactory.getLogger(BenchmarkDominanceResource.class);
+
+    @GET
+    @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML + Q, MediaType.TEXT_XML + Q})
+    public BenchmarkResult benchmark(@Context UriInfo uriInfo, @Context Request grizzlyRequest) {
+        List<Double> estimatedRW = new LinkedList<>();
+        List<Double> realRW = new LinkedList<>();
+        List<Long> time = new LinkedList<>();
+        Router router = null;
+        RoutingRequest request = null;
+        List<GraphPath> paths = null;
+        Random generator = new Random(1);
```

Comment:
seed powinien być parametrem zapytania

Reply:
Good idea

Reply:
Good idea ale trudno wykonywalne. Dla uproszczenia korzystam z tego samego requesta co /route. Zmiana tego to dużo zachodu a dodanie seeda do RoutingOptions wydaje mi się zły rozwiązaniem. Dla świętego spokoju proponuję to zostawić, raczej nikt nigdy nie skorzysta z seeda.